assets/facets.js

```diff
@@ -8,6 +8,12 @@ class FacetFiltersForm extends HTMLElement {
     }, 500);
 
     this.querySelector('form').addEventListener('input', this.debouncedOnSubmit.bind(this));
+    this.querySelector('form').addEventListener('keydown', (event) => { 
```

Comment:
Do you mean that instead of doing the above id do something like saving it in a variable (pseudo code below)? 🤔 Trying to make sure I understand

`const allElements = this.querySelectorAll('li')`

```
    allElements.addEventListener('keydown', (event) => { 
      if (event.target.tagName === 'LI' && event.code === 'Space') {
        event.preventDefault();
        event.target.querySelector('input').click();
      }
    });
```